lib/index.js

```diff
@@ -216,33 +216,35 @@ exports.read = function (res, options, callback) {
         res.removeListener('close', onResClose);
         res.on('error', Hoek.ignore);
 
-        if (err ||
-            !options.json) {
-
+        if (err || !options.json) {
             return callback(err, buffer);
         }
 
         // Parse JSON
 
-        var contentType = (res.headers && res.headers['content-type']) || '';
-        var mime = contentType.split(';')[0].trim().toLowerCase();
-
-        if (mime !== 'application/json') {
-            return callback(null, buffer);
-        }
+        var result;
+        var mode = (options.json === true || options.json.toLowerCase() === 'smart') ? 'smart' : 'force';
```

Comment:
doesn't seem like you need this line, could check `options.json === 'force'` directly on line 232